src/mds/topology/topology_item.h

```diff
@@ -510,7 +511,8 @@ class ChunkServer {
           startUpTime_(0),
           status_(status),
           onlineState_(onlineState),
-          dirty_(false) {}
+          dirty_(false),
+          version_(std::move(version)) {}
```

Comment:
the order of constructor initializer list must be strict with the order of data definition